data_dir/mv_synchronous_updates.yaml

```diff
@@ -0,0 +1,87 @@
+# LWT test: create and update data using LWT.
+### DML ###
+
+# Keyspace Name
+keyspace: mv_synchronous_ks
+
+# The CQL for creating a keyspace (optional if it already exists)
+keyspace_definition: |
+  CREATE KEYSPACE IF NOT EXISTS mv_synchronous_ks WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};
+
+# Table name
+table: mv_synchronous_cf
+
+# The CQL for creating a table you wish to stress (optional if it already exists)
+table_definition: |
+  CREATE TABLE IF NOT EXISTS mv_synchronous_cf (
+        domain int,
+        published_date int,
+        mv_pk int,
+        url text,
+        author text,
+        title text,
+        PRIMARY KEY(domain)
+  ) WITH comment='A table to hold blog posts'
+
+extra_definitions:
+  - create MATERIALIZED VIEW cf_update_2_columns_mv_pk as select mv_pk, author from mv_synchronous_cf where domain is not null and mv_pk > 0 and mv_pk <= 100000000 PRIMARY KEY(mv_pk, domain) WITH synchronous_updates = true;
+  - create MATERIALIZED VIEW cf_update_mv_pk as select * from mv_synchronous_cf where domain is not null and mv_pk > 100000001 and mv_pk < 200000000 PRIMARY KEY(mv_pk, domain) WITH synchronous_updates = true;
+  - create MATERIALIZED VIEW cf_update_asynch_test as select author from mv_synchronous_cf where domain is not null and published_date >= 0 and published_date < 5000000 PRIMARY KEY(published_date, domain);
+  - create MATERIALIZED VIEW cf_update_synch_test as select title from mv_synchronous_cf where domain is not null and published_date > 5000001 and published_date < 10000000 PRIMARY KEY(published_date, domain) WITH synchronous_updates = true;
+  - create MATERIALIZED VIEW cf_update_asynch_view_1 as select author from mv_synchronous_cf where domain is not null and published_date is not null PRIMARY KEY(published_date, domain);
+  - create MATERIALIZED VIEW cf_update_asynch_view_2 as select author from mv_synchronous_cf where domain is not null and published_date is not null PRIMARY KEY(published_date, domain);
+  - create MATERIALIZED VIEW cf_update_asynch_view_3 as select author from mv_synchronous_cf where domain is not null and published_date is not null PRIMARY KEY(published_date, domain);
+  - create MATERIALIZED VIEW cf_update_synch_view_1 as select title from mv_synchronous_cf where domain is not null and published_date is not null PRIMARY KEY(published_date, domain) WITH synchronous_updates = true;
+  - create MATERIALIZED VIEW cf_update_synch_view_2 as select title from mv_synchronous_cf where domain is not null and published_date is not null PRIMARY KEY(published_date, domain) WITH synchronous_updates = true;
+  - create MATERIALIZED VIEW cf_update_synch_view_3 as select title from mv_synchronous_cf where domain is not null and published_date is not null PRIMARY KEY(published_date, domain) WITH synchronous_updates = true;
```

Comment:
Same here, I don't understand why do we have 3 same views

Reply:
I added few MVs to create more pressure, perform more MV updates